code/game/objects/items/bodybag.dm

```diff
@@ -84,6 +84,31 @@
 	else
 		icon_state = icon_opened
 
+/obj/structure/closet/body_bag/open()
+	var/mob/living/L = locate(/mob/living) in src
+	var/mob/user = usr
```

Comment:
What I mean is that I want you to go into closets.dm and pass user as an arg anywhere open() or close() gets called

If there are places open() and close() aren't called by a person that shows you why this use of usr is a really bad idea (and why you need a sanity check too)